source/hardware/controllers/pioneer_ddj_200.rst

```diff
@@ -71,16 +72,19 @@ Mixer section (p. 10)
    :alt: Pioneer DDJ-200 (mixer section)
    :figclass: pretty-figures
 
-   Pioneer DDJ-200 (mixer section)
+   Mixer section (p. 10)
 
 ===  =================================================  ============================================================================================
 No.  Control                                            Function
 ===  =================================================  ============================================================================================
-1    :hwlabel:`MASTER CUE` button                       Activates Master output on Headphones
-2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs  Adjust the high/mid/low-frequency regions of the song.
+1    :hwlabel:`MASTER CUE` button                       Toggle Master/Cue knob of headphones between left and right
+1    :hwlabel:`SHIFT` + :hwlabel:`MASTER CUE` button    Toggle between 2- and 4-deck mode
+2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs  Adjust high/mid/low-frequencies
 3    :hwlabel:`CFX` knobs                               Turns on the selected effects
-4    :hwlabel:`HEADPHONE CUE` button                    Toggle PFL for the Channel
-5    Channel faders                                     Adjust the output level for each channel.
-6    Transition FX Button                               Turns on :ref:`AutoDJ <djing-auto-dj>`.
-7    Cross fader                                        Fades between left and right deck.
+4    :hwlabel:`HEADPHONE CUE 1 / 2` button              Toggle headphone pre-fader listening of channel 1 / 2, in 4-deck mode LED indicate deck 3 and 4
+4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1 / 2`    Load selected track to left or right selected deck
+4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1 / 2`   In 4-deck mode: toggle between deck 1 / 3 and 2 / 4, LED indicate deck 3 and 4
+5    Channel faders                                     Adjust the output level for each channel
+6    Transition FX Button                               Turns on :ref:`AutoDJ <djing-auto-dj>`
+7    Cross fader                                        Fade between left and right deck
```

Comment:
Please align the table columns properly, otherwise the manual will refuse to build. I can't post a suggestion though because GitHub refuses to let me.

Reply:
I aligned the mixer table columns.